features/step_definitions/sd-cmd.js

```diff
@@ -0,0 +1,274 @@
+                    });
+                });
+        });
+
+    Given(/^(.+) command is not exists yet$/,
+          { timeout: TIMEOUT }, function step(command) {
+            this.command = command;
+
+            return request({
+                uri: `${this.instance}/${this.namespace}/commands/${this.commandNamespace}`
+                + `/${this.command}`,
+                method: 'GET',
+                json: true,
+                auth: {
+                    bearer: this.jwt
+                }
+            })
+                .then((response) => {
+                    if (response.statusCode === 404) {
+                        return;
+                    }
+
+                    return request({
+                        uri: `${this.instance}/${this.namespace}/commands/${this.commandNamespace}`
+                        + `/${this.command}`,
+                        method: 'DELETE',
+                        json: true,
+                        auth: {
+                            bearer: this.jwt
+                        }
+                    }).then((response) => {
+                        Assert.equal(response.statusCode, 204);
+                    });
+                });
+          });
+
+    Given(/^"([^"]+)" version of the command is uploaded in "([^"]+)" tag$/, {
+        timeout: TIMEOUT
+    }, function step(version, tag) {
+        const jobName = `publish-${tag}`;
+        return request({
+            uri: `${this.instance}/${this.namespace}/pipelines/${this.pipelineId}/jobs`,
+            method: 'GET',
+            json: true,
+            auth: {
+                bearer: this.jwt
+            }
+        })
+            .then((response) => {
+                Assert.equal(response.statusCode, 200);
+
+                for (let i = 0; i < response.body.length; i += 1) {
+                    if (response.body[i].name === jobName) {
+                        this.jobId = response.body[i].id;
+                        return;
+                    }
+                }
+            })
+            .then(() =>
+                request({
+                    uri: `${this.instance}/${this.namespace}/builds`,
+                    method: 'POST',
+                    body: {
+                        jobId: this.jobId
+                    },
+                    auth: {
+                        bearer: this.jwt
+                    },
+                    json: true
+                }).then((response) => {
+                    Assert.equal(response.statusCode, 201);
+
+                    return this.waitForBuild(response.body.id).then((response) => {
+                        Assert.equal(response.statusCode, 200);
+                        Assert.equal(response.body.status, 'SUCCESS');
+                    });
+                })
+            );
+      });
+
+    When(/^execute (.+) job$/, { 
+        timeout: TIMEOUT
+    }, function step(jobName) {
+        return request({
```

Comment:
Instead of doing these requests, can we switch to use the event endpoints for creating new builds? just to be consistent with other functional tests and we may deprecate the build endpoint in the future.

https://github.com/screwdriver-cd/screwdriver/blob/master/features/step_definitions/secrets.js#L48

Reply:
Thaks, I'll fix it.